components/director/internal/domain/formationassignment/service_test.go

```diff
@@ -33,6 +35,9 @@ var (
 	testErr = errors.New("Test Error")
 
 	faInput = fixFormationAssignmentModelInput(TestConfigValueRawJSON)
+	fa      = fixFormationAssignmentModelWithFormationID("test")
```

Comment:
you could use `TestFormationID` here. Also for the rest of the changes we could check and use already introduced test variableas and if we don't have them - we can create them